src/request.rs

```diff
@@ -205,6 +245,39 @@ mod tests {
         db::delete_secret(&secret_id).unwrap();
     }
 
+    #[test]
+    fn test_secret_bundle() {
+        let secret_id = Uuid::new_v4().to_hyphenated().to_string();
+        let bundle_id = Uuid::new_v4().to_hyphenated().to_string();
+        let guid = "2cf13667-ea72-4013-9dd6-155e89c5a28f".to_string();
+        let request = RequestDetails {
+            guid: guid.clone(),
+            format: "json".to_string(),
+            secret_type: "bundle".to_string(),
+            id: bundle_id.clone(),
+        };
+
+        let secret_value = "dGVzdCBzZWNyZXQ=".to_string(); // "test secret" -> b64
```

Comment:
Nit: I'd consider doing this "long hand" for clarity. It also means you can drop the comment which cannot get out of sync with the code:

```suggestion
        let secret = "test secret";
        let secret_value = base64::encode(secret);
```